project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -87,6 +87,12 @@
                        attrs="{'required':[('resource_type','=','equipment')], 'readonly': [('is_task_event','=',True)]}"
                        widget="many2many_tags"/>
             </field>
+
```

Comment:
if you place client_type in the project_event module add this field in the projet_event/views/calendar_event_category_view.xml when